glados/tts.py

```diff
@@ -214,9 +214,10 @@ class Synthesizer:
         Converts the given phonemes to audio.
     """
 
-    def __init__(self, model_path: str, use_cuda: bool):
+    def __init__(self, model_path: str, use_cuda: bool, espeak: str):
         self.session = self._initialize_session(model_path, use_cuda)
         self.id_map = PHONEME_ID_MAP
+        self._espeak = espeak
```

Comment:
Specifically I'd move the auto-detection here - check if the `espeak` param is empty or not, if it is, auto-detect

Reply:
Yes, that's a good idea. I have updated the PR.  Would make sense to squash the two commits